updates_and_signals/atomic_message_handlers/starter.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import asyncio
+import logging
+import uuid
+from typing import Optional
+
+from temporalio import client, common
+from temporalio.client import Client, WorkflowHandle
+
+from updates_and_signals.atomic_message_handlers.workflow import (
+    ClusterManagerInput,
+    ClusterManagerWorkflow,
+)
+
+
+async def do_cluster_lifecycle(wf: WorkflowHandle, delay_seconds: Optional[int] = None):
+    allocation_updates = []
+    for i in range(6):
+        allocation_updates.append(
+            wf.execute_update(
+                ClusterManagerWorkflow.allocate_n_nodes_to_job, args=[f"task-{i}", 2]
+            )
+        )
+    await asyncio.gather(*allocation_updates)
+
+    if delay_seconds:
+        await asyncio.sleep(delay_seconds)
+
+    deletion_updates = []
+    for i in range(6):
+        deletion_updates.append(
+            wf.execute_update(ClusterManagerWorkflow.delete_job, f"task-{i}")
+        )
+    await asyncio.gather(*deletion_updates)
+
+    await wf.signal(ClusterManagerWorkflow.shutdown_cluster)
```

Comment:
Arguably shutdown could be an update that returns what the workflow returns instead of making it a two-step process (but this is fine too)

Reply:
Cool idea, and would show off the power of update.  Ran out of time this A.M, though.